spec/spec_helper.rb

```diff
@@ -35,6 +39,7 @@ class Post < ActiveRecord::Base; end
   ActiveRecord::Migration.create_table :posts do |t|
     t.string :author
     t.string :content
+    t.datetime :created_at
```

Comment:
What about adding two test cases, one for timestamps and one for a datetime attribute? They are slightly different and would better reflect a real world scenario
```suggestion
    t.timestamps
    t.datetime :some_other_datetime_field
```